### DISCUSSION TITLE
Smart Card Support

### DISCUSSION THREAD

**Discussion by gwoodw435** [usnistgov/NEMO#247] [General]
Does the NEMO support a smart card authentication? How do I contact the NEMO developer for inquiry such as the Docker installation on local PC and integrate it with the smart card? Thanks.

**Comment by rptmat57** [comment]
Hi,
NEMO support smart card authentication via badge reader for the Kiosk and Area access pages.
If you are talking about web application authentication, NEMO doesn't do authentication itself but you can setup NEMO with Shibboleth or OKTA or other authentication providers that can be configured to accept PIV/Smart cards.

For Docker installation you can consult the wiki: https://github.com/usnistgov/NEMO/wiki

**Reply by gwoodw435** [reply]
Hi. Thank you for a reply. Yes, it is for the web application authentication. Ok, I will investigate the authentication provider the agency is using. 

One more question, I have successfully installed the NEMO with the Docker on my Mac computer. Will it be able to configure with the authentication provider on my Mac? Or do I have to do the Development setup instead of the "Installation with Docker"? 

Thanks,
Greg

**Comment by rptmat57** [comment]
You should be able to configure it without doing the development setup, but it will be a bit harder to troubleshoot if you have any issues.
